### ISSUE TITLE
Improve prebuild-in-progress page layout and actions as well as make workspace start sequence clearer

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/gitpod#10041]
### Problem to solve

Following up from https://github.com/gitpod-io/gitpod/pull/9904#discussion_r871268558, the **prebuild-in-progress** phase:
1. Happens before the workspace loading phases (Checking, Creating, & Starting), however, it's not clear if it takes place before or after the rest of the workspace start phases.
2. Does not re-use the prebuild log output component from the prebuild page, see https://github.com/gitpod-io/gitpod/issues/9132.
3. Currently hijacks the title element from the existing loading phases of the workspace start page to indicate it's about the prebuild-in-progress phase.
4. Does not feel connected with the rest of the workspace start phases.

### Proposal

Since the existing loading phases of the workspace start page aim to abstract the sub-states of the loading cycle to make the loading friendlier to the user, we could consider 🅰️ reusing the same element from the prebuilds pages (see https://github.com/gitpod-io/gitpod/issues/9132) where we could surface these sub-states of the prebuild phase and 🅱️ provide better visibility of the step sequence.

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="loading-0" src="https://user-images.githubusercontent.com/120486/168054711-fd0530ba-405f-421f-8811-8140cc4cc304.png"> | <img width="1440" alt="loading-5" src="https://user-images.githubusercontent.com/120486/168062587-be9cc612-0b66-4c29-ad86-36f82c9bd453.png"> |

In short, to break things down into smaller minimum viable changes (MVC) if needed:
1. Introduce a _Preparing_ loading phase for the prebuild-in-progress phase
2. Additionally, consider re-using the prebuild log output element for better associating prebuilds across the product as well as surface potential sub-states of the prebuild-in-progress phase as well as improve the UX of the actions available to users, see https://github.com/gitpod-io/gitpod/issues/9132.

**Comment by atduarte** [comment]
Just sharing my 2 cents as I saw this being shared: I like the additional info 🙌  although I'm concerned about reducing the width of the window with logs.

**Comment by gtsiolis** [comment]
Thanks for the feedback, @atduarte! The width aspect has been brought up before in https://github.com/gitpod-io/gitpod/issues/9132#issuecomment-1090827146. Keeping the existing width can be also a good way forward and will be possible without further design specs as the component is flexible to change its width. 🤝 

Re-posting from https://github.com/gitpod-io/gitpod/issues/9132#issuecomment-1092072758 for visibility:

> No strong feelings on the width of this element, we can safely move forward with the same width for this iteration. 🛹

> To provide some context, the idea of the smaller element width was based on the intent to provide a fun and futuristic aspect in the product (see relevant comments[[1](https://github.com/gitpod-io/gitpod/pull/3560#discussion_r601662022)][[2](https://github.com/gitpod-io/gitpod/pull/3560#discussion_r602086714)][[3](https://github.com/gitpod-io/gitpod/pull/3560#discussion_r602123684)]) while assuming that watching the prebuild logs is not much helpful most of the times during workspace start as there's nothing users can do besides continuing with the default image. Cc @jankeromnes because https://github.com/gitpod-io/gitpod/pull/3560.

> This idea comes with the intent to have a horizontal scrollable container or keeping the text wrapping of the logs as is, see [example prebuild](https://gitpod.io/t/gitpod/gitpod/f656a014-0ceb-49c7-afb1-0220f9c11222) in gitpod-io/gitpod project, while focusing on the visibility of the prebuild task completion output.

> To support this idea, there was an early design draft we made for the onboarding flow which allowed users to expand the logs output element in width and height while using a narrower width by default which could be re-used here, see arrow on the top right corner in the exports below.

> | Logs collapsed | Logs expanded | Prebuild task |
> |-|-|-|
> | <img width="1440" alt="running-collapsed" src="https://user-images.githubusercontent.com/120486/162267049-913f3e8b-451c-4cc7-a1e6-46c4dc20c3ac.png"> | <img width="1440" alt="running-expanded" src="https://user-images.githubusercontent.com/120486/162267057-80feef29-0c58-4799-8178-efe5d1c96e7c.png"> | <img width="472" alt="Frame 371" src="https://user-images.githubusercontent.com/120486/162271812-ec04aa8c-6fac-4fcd-812a-f67bac08bf8f.png"> |
